src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -12,10 +12,11 @@
 # permissions and limitations under the License.
 
 from typing import Callable, Optional, List, Tuple
-
+from pathlib import Path
 import pandas as pd
-from autogluon import TabularPrediction as task
+from autogluon import tabular
```

Comment:
change to: `from autogluon.tabular import TabularPredictor`

Reply:
We have a TabularPredictor in gluon-ts also. Doing that might result in name conflict. See [here](https://github.com/awslabs/gluon-ts/blob/bee033d964951eb632414153d8ba32f0f2f3718c/src/gluonts/nursery/autogluon_tabular/predictor.py#L99)

Reply:
If we really need to make a distinction in the names, the GluonTS class could be called AutogluonTabularPredictor, indicating that it is a predictor based on Autogluon. The previously proposed name (GluonTSTabularPredictor) was redundant in my opinion

Reply:
Yes I vote for:
`from autogluon.tabular import TabularPredictor as AutogluonTabularPredictor`